server/src/main/java/org/opensearch/action/admin/indices/alias/TransportIndicesAliasesAction.java

```diff
@@ -200,7 +200,7 @@ protected void masterOperation(
         request.aliasActions().clear();
         IndicesAliasesClusterStateUpdateRequest updateRequest = new IndicesAliasesClusterStateUpdateRequest(unmodifiableList(finalActions))
             .ackTimeout(request.timeout())
-            .masterNodeTimeout(request.masterNodeTimeout());
+            .masterNodeTimeout(request.clusterManagerNodeTimeout());
```

Comment:
Looks like the `ClusterStateUpdateRequest` parent class method needs to be updated to be `clusterManagerNodeTimeout` ?

Reply:
Sure, I will rename it in issue #3544